elizaos-ai-configs/codex/CODEX.md

```
@@ -0,0 +1,123 @@
+# ElizaOS Development with OpenAI Codex
+
+## Project Overview
+
+ElizaOS V2: Multi-agent framework with cross-platform presence, blockchain management, autonomous workflows.
+
+**Repository:** https://github.com/elizaos/eliza
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)